doc/services/sensing/index.rst

```diff
@@ -0,0 +1,282 @@
+and a scheduling framework over sensor object relationships.
+Physical sensors do not depend on any other sensor objects for input, and
+will directly interact with existing zephyr sensor device drivers.
+Virtual sensors rely on other sensor objects (physical or virtual) as
+report inputs.
+
+The sensing subsystem relies on existing Zephyr sensor device APIs or V2
+zephyr sensor device APIs (`PR#44098 <https://github.com/zephyrproject-rtos/zephyr/pull/44098>`_),
+leveraging Zephyr's large library of sensor device drivers (100+).
+
+Use of the sensing subsystem is optional. Applications that only need to access simple sensors
+devices can use the Zephyr :ref:`sensor_api` API directly.
+
+Since the sensing subsystem is separated from device driver layer or
+kernel space and could support various customizations and sensor
+algorithms in user space with virtual sensor concepts. The existing
+sensor device driver can focus on low layer device side works, can keep
+simple as much as possible, just provide device HW abstraction and
+operations etc. This is very good for system stability.
+
+The sensing subsystem is decoupled with any sensor expose/transfer
+protocols, the target is to support various up-layer frameworks and
+Applications with different sensor expose/transfer protocols,
+such as `CHRE <https://github.com/zephyrproject-rtos/chre>`_, HID sensors Applications,
+MQTT sensor Applications according different products requirements. Or even support multiple
+Applications with different up-layer sensor protocols at the same time
+with it's multiple clients support design. For example can support CHRE
+and other normal zephyr sensor application (can use HID etc) at
+the same time.
+
+Sensing subsystem can help build a unified Zephyr sensing architecture for
+cross host OSes support and as well of IoT sensor solutions.
+
+The diagram below illustrates how the Sensing Subsystem integrates with up-layer frameworks.
+
+.. image:: images/sensing_solution.png
+   :align: center
+   :alt: Unified Zephyr sensing architecture.
+
+Configurability
+===============
+
+* Reusable and configurable standalone subsystem.
+* Based on Zephyr existing low-level Sensor API (reuse 100+ existing sensor device drivers)
+* Provide Zephyr high-level Sensing Subsystem API for Applications.
+* Separate option CHRE Sensor PAL Implementation module to support CHRE.
+* Decoupled with any host link protocols, it's Zephyr Application's role to handle different
+  protocols (MQTT, HID or Private, all configurable)
+
+Main Features
+=============
+
+* Scope
+    * Focus on framework for sensor fusion, multiple clients, arbitration, data sampling, timing
+      management and scheduling.
+
+* Sensor Abstraction
+    * ``Physical sensor``: interacts with Zephyr sensor device drivers, focus on data collecting.
+    * ``Virtual sensor``: relies on other sensor(s), ``physical`` or ``virtual``, focus on
+                          data fusion.
```

Comment:
This bullet rendered with bold.  I think changing the indent of continuation line may fix it:
```suggestion
    * ``Virtual sensor``: relies on other sensor(s), ``physical`` or ``virtual``, focus on
      data fusion.
```

Reply:
Fixed, thanks a lot!